cypress/components/button-bar/button-bar.cy.tsx

```diff
@@ -148,27 +153,72 @@ context("Test for Button-Bar component", () => {
     });
   });
 
-  it("has the expected border radius styling when major button children passed", () => {
-    CypressMountWithProviders(<ButtonBarCustom />);
-
-    buttonDataComponent()
-      .eq(0)
-      .should("have.css", "border-radius", "32px 0px 0px 32px");
-    buttonDataComponent().eq(1).should("have.css", "border-radius", "0px");
-    buttonDataComponent()
-      .eq(2)
-      .should("have.css", "border-radius", "0px 32px 32px 0px");
-  });
-
-  it("has the expected border radius styling when minor button children passed", () => {
-    CypressMountWithProviders(<ButtonBarWithMinorButtonChildren />);
-
-    buttonDataComponent()
-      .eq(0)
-      .should("have.css", "border-radius", "4px 0px 0px 4px");
-    buttonDataComponent().eq(1).should("have.css", "border-radius", "0px");
-    buttonDataComponent()
-      .eq(2)
-      .should("have.css", "border-radius", "0px 4px 4px 0px");
+  describe("renders with ButtonMinor children", () => {
+    const buttonTypesAndBackgrounds = [
+      ["1st", "primary", 0, "rgb(162, 44, 59)"],
+      ["2nd", "secondary", 1, "rgba(0, 0, 0, 0)"],
+      ["3rd", "tertiary", 2, "rgba(0, 0, 0, 0)"],
+    ];
+
+    describe("renders with ButtonMinor children", () => {
+      it.each(buttonTypesAndBackgrounds)(
+        "should apply correct background-color on hover for %s button rendered with ButtonMinor children",
```

Comment:
Currently this outputs as `should apply correct background-color on hover for 1st button rendered with ButtonMinor children`
 If we update it slightly it will be a little clearer which specific tests have failed, the below suggestion will output
 
 `should apply correct background-color on hover for 1st button rendered with primary ButtonMinor children
 `

```suggestion
        "should apply correct background-color on hover for %s button when %s ButtonMinor children passed",
```